tests/types/src/invalid.ts

```diff
@@ -439,3 +439,19 @@ stripe.createPaymentMethod({
     },
   },
 });
+
+stripe
+  .createConfirmationToken({
+    elements,
+    params: {
+      // @ts-expect-error payment_method is not a valid parameter
+      payment_method: 'pm_12345',
+    },
+  })
+  .then((result) => {
+    if (result.error) {
+      return console.log(result.error.code);
+    }
```

Comment:
we should probably remove this console.log?

Reply:
The file doesn't seem to be run and a bunch of console.logs are throughout the file, so I was just copying how others were doing it.

Reply:
hmm i could see some in `tests/types/src/valid.ts` but not here. I think we should just remove console.log here and keep in the file?

Reply:
Yeah, i'll remove it then